tests/dummy/app/adapters/-json-api.js

```diff
@@ -1,4 +1,5 @@
 import EmberObject from '@ember/object';
+import fetch from 'fetch';
```

Comment:
what does this have to do with IE testing?

Reply:
I think we need it so it polyfills for IE11?

https://caniuse.com/fetch